magefile.go

```diff
@@ -2243,3 +2244,110 @@ func hasCleanOnExit() bool {
 	b, _ := strconv.ParseBool(clean)
 	return b
 }
+
+type dependency struct {
+	Name    string
+	Version string
+}
+
+func (d dependency) Clean(sep string) dependency {
+	cleanFn := func(dep, sep string) string {
+		chunks := strings.SplitN(dep, sep, 2)
+		if len(chunks) == 2 {
+			return chunks[1]
+		}
+
+		return dep
+	}
+
+	return dependency{
+		Name:    cleanFn(d.Name, sep),
+		Version: d.Version,
+	}
+}
+
+func OtelReadme() error {
```

Comment:
Can't say as of now if we'll end up with more OTel-specific mage targets, but if I had to guess I would say we would. So it might be a good idea to namespace all OTel-specific targets with `otel:`?  So this target would then become `otel:readme`.